sdk/nodejs/resource.ts

```diff
@@ -208,6 +208,10 @@ export interface ResourceOptions {
      * When set to true, protect ensures this resource cannot be deleted.
      */
     protect?: boolean;
+    /**
+     * Ignore changes to any of the specified properties.
+     */
+    ignoreChanges?: string[];
```

Comment:
I noticed that you added this to `ResourceOptions` instead of `CustomResourceOptions`.  What does `ignoreChanges` mean on component resource, since there are no CRUD operations associated with components?

Reply:
Technically, the feature does apply to both `Component` and `Custom` resources, so I think this is technically correct.  That said, I'm not sure practically what impact it can have to ignore changes on a component (though in the UI and checkpoint, this will have the effect of ignoring input property changes). But I think that's more about some of the current oddness around input/output properties on components generally.  I think for orthogonality purposes, it does make sense to keep this as is.

Reply:
:+1: